docs/transport/braginskii.rst

```diff
@@ -0,0 +1,60 @@
+.. _braginskii:
+
+Braginskii
+**********
+
+The `~plasmapy.physics.transport.braginskii` subpackage includes a 
+class interface 
+`~plasmapy.physics.transport.braginskii.ClassicalTransport` with 
+multiple methods which are involved in modeling classical transport.
+
+.. topic:: Examples:
+
+   * :ref:`sphx_glr_auto_examples_plot_braginskii.py`
+
+.. _Classical transport models:
+
+Classical transport models
+==========================
+
+Broad overview of classical transport models implemented within 
+`braginskii.py`. These include:
+Braginskii
+Spitzer-Harm
+Epperlein-Haines (under construction)
+Ji-Held
+
+.. _Classical-transport-class:
+
+Classical transport class
+=========================
+
+Explain how to use the 
+`~plasmapy.physics.transport.braginskii.ClassicalTransport` interface.
+
+.. _Conductivities-resistivities:
+
+Conductivities and Resistivities
+================================
+
+Explain:
+resistivity
```

Comment:
Would you mind if I put these notes of yours in close to verbatim? They're awesome.

I also think it'd be best to put these docs into the methods themselves... I'm going to check if I can access method docstrings from the wrapper functions to avoid code duplication.